spec/decorators/provider_decorator_spec.rb

```diff
@@ -1,6 +1,26 @@
 require "rails_helper"
 
 RSpec.describe ProviderDecorator do
+  describe "#partner_provider_placements" do
```

Comment:
Just had a thought. This should be `partner_school_placements`, because the model at the base of this is the provider. Would you mind renaming the method?